media-sound/mumble/mumble-9999.ebuild

```diff
@@ -49,7 +50,7 @@ RDEPEND="
 	jack? ( virtual/jack )
 	!libressl? ( >=dev-libs/openssl-1.0.0b:0= )
 	libressl? ( dev-libs/libressl )
-	opus? ( >=media-libs/opus-1.0.1 )
+	>=media-libs/opus-1.3.1
```

Comment:
Move this into the non-conditional block above. Keep it alphabetically sorted please.